browse/services/documents/fs_implementation/legacy_fs_paths.py

```diff
@@ -2,12 +2,11 @@
 
 from typing import Optional, List, Dict
 
-from browse.services.anypath import to_anypath, APath
+from arxiv.files.anypath import to_anypath, APath
```

Comment:
fs_implementation was written before the ObjectStore stuff. 

In the future we could get rid of cloudlibpath and anypath and just use ObjectStore.